next.js/src/app/company/productlist/components/RegisterFunc.tsx

```diff
@@ -1,90 +1,37 @@
 "use client";
 
-import { useContext, useState } from "react";
-import { CompanyContext } from "@/context/company-context";
-import { Box, Button, Stack, TextField, Typography } from "@mui/material";
+import { useState } from "react";
+import { Box, Button, Stack } from "@mui/material";
 
-export const RegisterFunc = () => {
-  const companyContext = useContext(CompanyContext);
-
-  const [modelNum, setModelNum] = useState("");
-  const [productName, setProductName] = useState("");
-  const [price, setPrice] = useState<number>(0);
-  const [description, setDescription] = useState("");
-
-  if (!companyContext) return <Typography>Loading...</Typography>;
-
-  const { fetchMyProducts } = companyContext;
-
-  const handleSubmitProduct = async (e: React.FormEvent) => {
-    e.preventDefault();
-    try {
-      await fetch("http://localhost:3001/api/company/addproduct", {
-        method: "POST",
-        credentials: "include",
-        headers: { "Content-Type": "application/json" },
-        body: JSON.stringify({
-          model_number: modelNum,
-          name: productName,
-          price,
-          description,
-        }),
-      });
-
-      setProductName("");
-      setModelNum("");
-      setPrice(0);
-      setDescription("");
-      await fetchMyProducts();
-    } catch (err) {
-      console.error(err);
-    }
-  };
+import { AddCustomProduct } from "./AddCustomProduct";
+import { AddDefaultProduct } from "./AddDefaultProduct";
 
+export const RegisterFunc = () => {
```

Comment:
![image](https://user-images.githubusercontent.com/50566499/259609094-fc0e8c2f-5f88-4927-8bb1-7d160c2279c9.png)  & ![image](https://user-images.githubusercontent.com/50566499/259608918-eca53dd1-56ab-4014-b855-f4496d4318e8.png) 

このような命名が良いかと思いました！
```suggestion
export const RegisterSection = () => {

// RegisterFormsとかでもいいかも
```